gulpfile.js

```diff
@@ -347,11 +345,7 @@ function generateManifest() {
 }
 
 gulp.task('watch-manifest', function () {
-  gulp.watch(MANIFEST_SOURCE_FILES, batch(function (events, done) {
-    endOfStream(generateManifest(), function () {
-      done();
-    });
-  }));
+  gulp.watch(MANIFEST_SOURCE_FILES, { delay: 500 }, generateManifest);
```

Comment:
The use of `endOfStream` was redundant here because `generateManifest` returns a `Stream` and `gulp.watch` already knows how to watch a stream and mark a task as done when the stream finishes.